app/models/local_census_record.rb

```diff
@@ -1,8 +1,12 @@
 class LocalCensusRecord < ApplicationRecord
+  ALLOWED_DOCUMENT_TYPES = { dni: "1", passport: "2", residence_card: "3" }.freeze
```

Comment:
Hi @javierm, i was thinking about apply your refactoring suggestions but finally i prefer to leave for another PR because i found some weird things that make me doubt like this method of  `Signature` model [1]

[1] https://github.com/consul/consul/blob/master/app/models/signature.rb#L96

What does the "4" value means? 🤔 I guess this could be forgotten trash from a previous implementation that was removed from other document_types methods but was forgotten here

Reply:
That method is only used by the `in_census?` method, in the lines:

```ruby
response = CensusCaller.new.call(document_type, document_number, date_of_birth, postal_code)

if response.valid?
 # ...
```

So it was probably a typo, since every other place using `CensusCaller` doesn't use the `4` :thinking:.